_posts/2019-01-30-gcal-for-project-management.md

```diff
@@ -0,0 +1,79 @@
+
+**Schedule Consistency**. Another way to keep consistency with client projects is to schedule project calls around the same time of day or the same day of the week. For a client with once a week check-in calls, it could be helpful to schedule that check-in every Monday morning.
+
+For a client that requests a weekly demo call and a weekly status call, it could be helpful to have both calls, for example, in the afternoon, say 4pm. Obviously this is not always possible, but it can be helpful for the client and the team to get into a routine of having work ready to go at the same time of day or day of the week.
+
+**Breaks**. When balancing multiple client projects, it is also crucial to schedule breaks between calls. I aim to have at least 15 minutes, ideally half an hour, between client calls. This time is necessary to wind down one call, finish jotting down notes and have time to properly prepare for the next one.
+
+#### Scheduling Meetings & Calls
+
+Scheduling meetings and calls with team members and clients can be complicated, especially with teams that have different schedules and live in multiple time zones. There are a few [Google Calendar](https://www.google.com/calendar) features that make scheduling a breeze.
+
+**Suggested Times & Find a Time Feature**. When creating an event, these two features work to find an ideal time for your event, based on the calendars of your event guests. To use them, all you need to do is create a new event and add your guests on the right side of the screen.
+
+First, the **Suggested Times** feature. You will find this button underneath the list of guests invited to your event. It picks a few times around your event date that would work for all of your guests.
+
+  <img src="/blog/assets/images/suggested-times.png" alt="Suggested Times Feature">
+
+Second, the **Find a Time** feature. You will find this button next to "Event Details". It shows a view of the calendars of each of your invited guests, allowing you to pick a time that you think would work best for everyone.
+
+  <img src="/blog/assets/images/find-a-time.png" alt="Find a Time Feature">
+
+**Adding a Call Link**. At [Ombu Labs](https://www.ombulabs.com), we generally work remotely, so it is important that every event created includes a video call link. Our preferred video call service is [Zoom](https://zoom.us). [Zoom](https://zoom.us) has a feature that allows you to schedule a video call and embed the link directly into a [Google Calendar](https://www.google.com/calendar) event. They also allow you to copy the video call link and paste it yourself into a pre-existing [Google Calendar](https://www.google.com/calendar) event.
```

Comment:
@ercohen14 I think that adding the link for `Zoom` only the first time you mention it is enough. (I noticed the same with other words like `Google Calendar` or `Ombu Labs`)